sql/src/main/java/io/crate/exceptions/ExpiredLicenseStatementNotAllowedException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.exceptions;
+
+public class ExpiredLicenseStatementNotAllowedException extends ValidationException implements ClusterScopeException {
```

Comment:
I added more info in the exception message, I did not however include the statement that failed as I cannot find a straightforward way of of getting this information of an `AnalyzedStatement`. Solutions I though of 
a) postponing the throw exception until the `visitX()` methods but did not like that as it would pollute the code imo
b) introducing a dedicated visitor for that but I though it is not worth it since:

the only information that the client needs to know is that the license is expired and he/she needs to purchase a new one. The read only queries we allow is for the admin UI, not for the client.

Thus, imo, no more is needed in the exception message

Happy to hear your thoughts on this